### PR TITLE
Cache all service entity in memory for query

### PR DESCRIPTION
**Pull request by mrproliu**
Querying all service entities and storing them in the cache periodically. All service-level queries should be modified to retrieve data from the cache.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleConfig.java [hunk 1]
```diff
@@ -217,6 +217,20 @@ public class CoreModuleConfig extends ModuleConfig {
...
+    @Setter
+    @Getter
+    private int serviceCacheRefreshInterval = 10;
+
+    /**
+     * The max size of service cache, default 10000
+     */
+    @Setter
+    @Getter
+    private int serviceCacheMaxSize = 10_000;
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleConfig.java:232]:
```suggestion
    private int serviceCacheMaxSize = 100_000;
```

As it is only a name cache,  a little more larger should be fine.

Comment (wu-sheng) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleConfig.java:232]: @kezhenxu94 As scrolling is there, this size should be fine, right?

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/menu/UIMenuManagementService.java [hunk 2]
```diff
@@ -48,7 +49,7 @@ public class UIMenuManagementService implements Service, Runnable {
     private ModuleManager moduleManager;
     private CompletableFuture<List<MenuItem>> menuItemFuture;
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/menu/UIMenuManagementService.java:50]: Future is going to be changed through another PR, right?

Comment (mrproliu) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/menu/UIMenuManagementService.java:50]: Yes.

**Review by kezhenxu94 — DISMISSED** [review]

**Review by wu-sheng — DISMISSED** [review]

**Review by wu-sheng — DISMISSED** [review]

**Comment by mrproliu** [comment]
Based on the log prompt, it seems that there is a problem displaying all Kubernetes resource logs. I suspect that this issue might be related to the Kubernetes resources, so I need to debug the container. Let me switch to draft mode for now.

**Comment by mrproliu** [comment]
The previous E2E error occurred due to insufficient storage space in GHA, resulting in the inability to write ES data. To resolve this, the data write threshold configuration for ES was updated, leading to the recovery of GHA.

I think now it's ready to review. @wu-sheng @kezhenxu94

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**